.github/workflows/e2e-nvidia-l4-x1.yml

```
@@ -141,7 +141,22 @@ jobs:
           HF_TOKEN: ${{ secrets.HF_TOKEN }}
         run: |
           . venv/bin/activate
+          # set preserve to true so we can retain the logs
+          export PRESERVE=1
           ./scripts/e2e-ci.sh -m
+          
+          # we know that the file will be named something like f"/training_params_and_metrics_global{os.environ['RANK']}.jsonl" in python
+          # and we know that it will be written into a directory created by mktemp -d. 
```

small readability nit
```suggestion
          # and we know that it will be written into a directory created by `mktemp -d`
```

---

These little details are important so I'm glad you caught this.